src/chi-vue/src/views/SearchInputView.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div id="search-input-view">
+    <div style="max-width: 20rem;">
```

Comment:
Maybe you can define a local class for this

Reply:
Done, thanks ;)